### PR TITLE
Added animations and fixed background

### PR DESCRIPTION
**Pull request by Bibhuti05**
image
Screenshot from 2024-08-06 19-15-55
Screenshot from 2024-08-06 19-16-46

### CODE REVIEW

**Comment by aditya9940** [comment]
@Bibhuti05 take the latest pull and resolve the merge confict

**Comment by Bibhuti05** [comment]
fixed animations

File: src/component/vipCard/vipcard.module.css [hunk 1]
```diff
@@ -28,12 +28,6 @@ p {
 
 }
 
-@supports(-moz-appearance:none) {
```
Comment (arijeet-mukherjee) [inline src/component/vipCard/vipcard.module.css:31]: why it is removed ?

File: src/component/vipCard/vipcard.module.css [hunk 2]
```diff
@@ -152,7 +147,7 @@ p {
     position: absolute;
     inset: 0;
     z-index: -1;
-    background: linear-gradient(0deg, rgba(0, 0, 0, 0.9) 15%, rgba(255, 255, 255, 0.5) 100%);
```
Comment (arijeet-mukherjee) [inline src/component/vipCard/vipcard.module.css:155]: It is also updated why ?

**Pull request merged**